pyani_plus/private_cli.py

```diff
@@ -625,7 +625,11 @@ def log_dnadiff(
         subject_hash=subject_md5,
         identity=identity,
         aln_length=aligned_bases_with_gaps - gap_lengths,
-        sim_errors=None,  # Leaving this as None for now (How should we calculate this?)
+        sim_errors=int(
```

Comment:
Do we want to round down (which is what ``int`` will do), or round to nearest whole number (with ``round``)?

Reply:
I just used `int` for now, but since I'm not a mathematician not sure if `round` would be more appropriate. Perhaps, leaving this until monday won't do any harm.

Reply:
We should use `round()` to get the closest integer.